gateway/modules/schema/helpers.go

```diff
@@ -93,6 +93,9 @@ func (c *creationModule) addNotNull() string {
 	case utils.Postgres:
 		return "ALTER TABLE " + c.schemaModule.getTableName(dbType, c.logicalDBName, c.TableName) + " ALTER COLUMN " + c.ColumnName + " SET NOT NULL"
 	case utils.SQLServer:
+		if c.columnType == "varchar("+model.SQLTypeIDSize+")" || c.columnType == "varchar(max)" {
+			return "ALTER TABLE " + c.schemaModule.getTableName(dbType, c.logicalDBName, c.TableName) + " ALTER COLUMN " + c.ColumnName + " " + c.columnType + " collate Latin1_General_CS_AS NOT NULL"
+		}
```

Comment:
Whats the point of this? Maybe you should just check if there is a `varchar` prefix?

Reply:
yep

Reply:
Okay. So @prithvi081099 you'll need to change this everywhere

Reply:
I have made the changes @YourTechBud and @sharadregoti